File: src/main/java/nl/knaw/huc/api/AboutInfo.java
package nl.knaw.huc.api;

public class AboutInfo {
  private String appName;
  private String startedAt;

  public void setAppName(String appName) {
    this.appName = appName;
  }

  public String getAppName() {
    return appName;
  }

  public void setStartedAt(String startedAt) {
    this.startedAt = startedAt;
  }

  public String getStartedAt() {
    return startedAt;
  }
}

File: server/src/main/kotlin/nl/knaw/huc/annorepo/api/AboutInfo.kt
package nl.knaw.huc.annorepo.api

data class AboutInfo(
    val appName: String,
    val version: String,
    val startedAt: String,
    val baseURI: String,
    val source: String = "https://github.com/knaw-huc/annorepo",
    val needsAuthentication: Boolean
)